### DISCUSSION TITLE
First Things First🦀

### DISCUSSION THREAD

**Discussion by sajvn0x** [obhq/exfat#5] [General]
Can I get some reference links to learn **exFAT**?
I love to contribute to this repo. How to test the code you wrote

**Comment by ultimaweapon** [comment]
Sure. I was using the official specs: https://learn.microsoft.com/en-us/windows/win32/fileio/exfat-specification

I did not wrote any test cases yet (and a PR to add the test cases is welcome!). Actually this project was extracted from https://github.com/obhq/obliteration. So I only tested it manually when it is still a part of Obliteration project.

If you need help or have any questions just let me know.

**Reply by sajvn0x** [reply]
I don't know how to use Rust Properly to do projects, I have never done any projects using Rust. I think I can get some Rust + exFAT knowledge from this repo with your professional brain

**Reply by ultimaweapon** [reply]
I'll try my best to help you.
